### DISCUSSION TITLE
Graph is not showing through reverse proxy

### DISCUSSION THREAD

**Discussion by DastardlyDino** [firefly-iii/firefly-iii#7696] [Firefly III]
I am having an issue with graphs not showing up when accessing through a reverse proxy. When I go directly to the local IP locally I see all graphs just fine. It only happens when viewing from my subdomain. All the graphs show a warning/caution sign. Any Ideas on how to fix this?

This is the exact same issue as [chucklessducks back in 2021](https://github.com/firefly-iii/firefly-iii/issues/4504) . Unfortunately no one provided a solution and the discussion went stale. Does anyone know what causes this issues and how I can fix it?

![](https://user-images.githubusercontent.com/75915681/111051729-f509ac00-841a-11eb-8f16-db307c513aa3.PNG)

**Comment by JC5** [comment]
I still have to add this question to the Firefly III section 

https://docs.firefly-iii.org/data-importer/faq/installation-errors/#i-get-page-load-errors-because-the-protocols-dont-match

**Reply by DastardlyDino** [reply]
So while what you linked me was for the importer it made me realized I never looked at the install directions for Firefly III since I was using an Unraid Template. After looking at the install directions for Firefly III (not importer) I noticed I was missing the APP_URL=https://firefly.example.com tag and the TRUSTED_PROXIES=* tag in my template (https://docs.firefly-iii.org/firefly-iii/installation/docker/#docker-and-reverse-proxies). Now my graphs are now working perfectly with those tags added in. Thank you for your help.
